### ISSUE TITLE
npm install -g fails

### ISSUE THREAD

**Issue by joshterrill** [n1k0/casperjs#1260]
Here is the error I get after running an ```npm install -g casperjs``` as instructed on the home page.
```
0 info it worked if it ends with ok
1 verbose cli [ 'c:\\Program Files\\nodejs\\node.exe',
1 verbose cli   'c:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js',
1 verbose cli   'install',
1 verbose cli   '-g',
1 verbose cli   'casperjs' ]
2 info using npm@2.11.2
3 info using node@v0.12.5
4 verbose install initial load of C:\Users\Clear\AppData\Roaming\npm\package.json
5 verbose readDependencies loading dependencies from C:\Users\Clear\AppData\Roaming\npm\package.json
6 silly cache add args [ 'casperjs', null ]
7 verbose cache add spec casperjs
8 silly cache add parsed spec { raw: 'casperjs',
8 silly cache add   scope: null,
8 silly cache add   name: 'casperjs',
8 silly cache add   rawSpec: '',
8 silly cache add   spec: '*',
8 silly cache add   type: 'range' }
9 silly addNamed casperjs@*
10 verbose addNamed "*" is a valid semver range for casperjs
11 silly addNameRange { name: 'casperjs', range: '*', hasData: false }
12 silly mapToRegistry name casperjs
13 silly mapToRegistry using default registry
14 silly mapToRegistry registry https://registry.npmjs.org/
15 silly mapToRegistry uri https://registry.npmjs.org/casperjs
16 verbose addNameRange registry:https://registry.npmjs.org/casperjs not in flight; fetching
17 verbose request uri https://registry.npmjs.org/casperjs
18 verbose request no auth needed
19 info attempt registry request try #1 at 6:25:21 PM
20 verbose request id 1601ac79e4baebdf
21 verbose etag "8UB04TAHFSUT0SUGUN8IDRPIH"
22 http request GET https://registry.npmjs.org/casperjs
23 http 304 https://registry.npmjs.org/casperjs
24 silly get cb [ 304,
24 silly get   { date: 'Thu, 02 Jul 2015 01:25:15 GMT',
24 silly get     via: '1.1 varnish',
24 silly get     'cache-control': 'max-age=60',
24 silly get     etag: '"8UB04TAHFSUT0SUGUN8IDRPIH"',
24 silly get     age: '19',
24 silly get     connection: 'keep-alive',
24 silly get     'x-served-by': 'cache-lax1435-LAX',
24 silly get     'x-cache': 'HIT',
24 silly get     'x-cache-hits': '1',
24 silly get     'x-timer': 'S1435800315.234429,VS0,VE0',
24 silly get     vary: 'Accept' } ]
25 verbose etag https://registry.npmjs.org/casperjs from cache
26 verbose get saving casperjs to C:\Users\Clear\AppData\Roaming\npm-cache\registry.npmjs.org\casperjs\.cache.json
27 silly addNameRange number 2 { name: 'casperjs', range: '*', hasData: true }
28 silly addNameRange versions [ 'casperjs', [ '1.1.0-beta2', '1.1.0-beta3' ] ]
29 verbose stack Error: No compatible version found: casperjs@'*'
29 verbose stack Valid install targets:
29 verbose stack ["1.1.0-beta2","1.1.0-beta3"]
29 verbose stack
29 verbose stack     at installTargetsError (c:\Program Files\nodejs\node_modules\npm\lib\cache\add-named.js:274:12)
29 verbose stack     at next (c:\Program Files\nodejs\node_modules\npm\lib\cache\add-named.js:253:17)
29 verbose stack     at setData (c:\Program Files\nodejs\node_modules\npm\lib\cache\add-named.js:230:5)
29 verbose stack     at RES (c:\Program Files\nodejs\node_modules\npm\node_modules\inflight\inflight.js:23:14)
29 verbose stack     at f (c:\Program Files\nodejs\node_modules\npm\node_modules\once\once.js:17:25)
29 verbose stack     at fixName (c:\Program Files\nodejs\node_modules\npm\lib\cache\add-named.js:29:5)
29 verbose stack     at saved (c:\Program Files\nodejs\node_modules\npm\lib\cache\caching-client.js:173:7)
29 verbose stack     at FSReqWrap.oncomplete (fs.js:95:15)
30 verbose cwd c:\Projects\artemis
31 error Windows_NT 6.1.7601
32 error argv "c:\\Program Files\\nodejs\\node.exe" "c:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "-g" "casperjs"
33 error node v0.12.5
34 error npm  v2.11.2
35 error code ETARGET
36 error notarget No compatible version found: casperjs@'*'
36 error notarget Valid install targets:
36 error notarget ["1.1.0-beta2","1.1.0-beta3"]
36 error notarget
36 error notarget This is most likely not a problem with npm itself.
36 error notarget In most cases you or one of your dependencies are requesting
36 error notarget a package version that doesn't exist.
37 verbose exit [ 1, true ]
```

**Comment by hamishtaplin** [comment]
Same problem here. Anyone know how to fix this?

**Comment by tholu** [comment]
+1 on Debian

**Comment by nathanboktae** [comment]
Does `npm install casperjs@1.1.0-beta3` work?

**Comment by tholu** [comment]
@nathanboktae Yes, ```npm -g install casperjs@1.1.0-beta3``` works. Any idea why it doesn't work when the version is not explicitly specified?

**Comment by nathanboktae** [comment]
Likely a combination of npm trying to match node `0.12` and a non-beta version. @n1k0 any reason you haven't published a new Casper version in nearly two years?

**Comment by shobhitg** [comment]
This issue still happens.

I so dislike the idea of running `npm -g install casperjs@1.1.0-beta3` instead of `npm -g install casperjs`

Is there any update on this?

**Comment by istr** [comment]
We will address this issue while creating ```1.1.0```. Please don't ask for timelines, [as the current project maintenance skates on very thin ice](https://github.com/n1k0/casperjs/issues/1350).

**Comment by BIGjuevos** [comment]
@istr Is the goal to close this issue when 1.1.0 is tagged?

**Comment by istr** [comment]
@BIGjuevos I guess that this is already closed with the latest two published npm packages. Would you mind checking it?

**Comment by BIGjuevos** [comment]
Just checked it in a brand new ubuntu virtual host.  Works as expected on a fresh install.

    test@test-ubuntu:~$ sudo npm install casperjs -g
    /usr/local/bin/casperjs -> /usr/local/lib/node_modules/casperjs/bin/casperjs
    casperjs@1.1.0-beta5 /usr/local/lib/node_modules/casperjs

@istr I think we are good to resolve the issue.

**Comment by istr** [comment]
Closing. Seems resolved in 1.1.0-beta5. Thanks @BIGjuevos for validation.
